src/tests/fuzz/fuzz_predefpcap_aware.c

```diff
@@ -117,11 +117,11 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
     // loop over packets
     r = FPC_next(&pkts, &header, &pkt);
     p = PacketGetFromAlloc();
-    if (header.ts.tv_sec >= INT_MAX - 3600) {
+    if (r <= 0 || header.ts.tv_sec >= INT_MAX - 3600) {
         goto bail;
     }
     p->ts.tv_sec = header.ts.tv_sec;
-    p->ts.tv_usec = header.ts.tv_usec % 1000000;
+    p->ts.tv_usec = ((header.ts.tv_usec % 1000000) + 1000000) % 1000000;
```

Comment:
what is the trade off between simply rejecting invalid input vs fixing it up like this? Feel this kind of fixing up is getting rather convoluted

Reply:
I think reject instead of fixing will make fuzzing slower

Reply:
seems irrelevant as long as we run out of memory due to the corpus size within a few hours anyway

Reply:
So, do you want to reject the input ? (with inconsistent timestamps, instead of fixing them)